README.md

```diff
@@ -18,19 +18,92 @@ API gateway to Quicksilver (Message Broker) functionality.
 | `POST /campaign/signup`     | [Campaign signup](documentation/endpoints/campaign.md#campaign-signup)           | [campaign.signup.transactional](documentation/messages/campaign.signup.transactional.md)      |
 | `POST /campaign/reportback` | [Campaign report back](documentation/endpoints/campaign.md#campaign-report-back) | [campaign.report_back.transactional](documentation/messages/campaign.signup.transactional.md) |
 
-## Tests
+## Development
+### Requirements
+- [Node.js](https://nodejs.org/en/download/) v4.4
+- [Docker](https://www.docker.com/products/overview) with support
+  of Compose file [v2](https://docs.docker.com/compose/compose-file/#/versioning)
+  for running dependent services (like RabbitMQ).
 
-Master build status:
+### Installation
+1. Install dependencies `npm install`
+2. Create `config/local.js` file with following settings:
 
-[![wercker status](https://app.wercker.com/status/ee29a549f6d3b68e7b3cb101a7c9a943/s/master "wercker status")](https://app.wercker.com/project/bykey/ee29a549f6d3b68e7b3cb101a7c9a943)
+   ```js
+   module.exports = {
+      // Northstar.
+      northstar: {
+        apiBaseURI: 'https://northstar-qa.dosomething.org/v1/users',
+        apiKey: '{{ nothstar_access_key }}',
+      },
 
-Test coverage uses the following utilities:
-- [Mocha](https://www.npmjs.com/package/mocha)
-- [Should](https://www.npmjs.com/package/should)
-- [Supertest](https://www.npmjs.com/package/supertest)
+      // Phoenix.
+      phoenix: {
+       baseURI: 'https://staging.beta.dosomething.org/api/v1',
+       username: '{{ phoenix_api_user }}',
+       password: '{{ phoenix_api_pass }}',
+      },
+
+     // RabbitMQ.
+     amqp: {
+       management: {
+         username: 'dosomething',
```

Comment:
I don't see why not. That's an example of local setup for this app. 
This would setup connection to a local RabbitMQ service running in docker, which is configured with this username, password and vhost.